tests/Utilities/constants.py

```diff
@@ -18,6 +18,9 @@
 food1 = Food("apple", fooddate1.isoformat(), 3)
 food2 = Food("pear", fooddate2.isoformat(), 3)
 food3 = Food("banana", fooddate3.isoformat(), 2)
+food4 = Food("soda", fooddate1.isoformat(), 3)
+food5 = Food("apple", fooddate1.isoformat(), 3)
+food6 = Food("pumpkin", fooddate1.isoformat(), 3)
```

Comment:
Why did you need to create `food6` instead of just using `food4`?

Reply:
You're right, I got rid of `food6` and used `food4` instead.